composer.json

```
@@ -10,13 +10,13 @@
         }
     ],
     "require" : {
-        "php": ">=7.0",
+        "php": "^7.1, <7.4",
```

```suggestion
        "php": "^7.1",
```

should be fine. No need to limit PHP version from minor releases so for example if we don't need this library in the future and don't maintain it, someone would still be able to run it on 7.4, but without being tested on 7.4

---

Changed.